### PR TITLE
Show metrics as JSON

### PR DESCRIPTION
**Pull request by g4s8**
#425 - added `/.metrics` endpoint to show collected metrics JSON.

### CODE REVIEW

File: src/main/java/com/artipie/metrics/MetricSlice.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+import java.util.Map;
+import javax.json.Json;
+import javax.json.JsonArrayBuilder;
+import org.reactivestreams.Publisher;
+
+/**
+ * Slice with metrics JSON.
+ * @since 0.10
+ */
+public final class MetricSlice implements Slice {
```
Comment (olenagerasimova) [inline src/main/java/com/artipie/metrics/MetricSlice.java:46]: @g4s8 please add todo to create test for this slice

File: src/main/java/com/artipie/http/OptionalSlice.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+import java.util.function.Supplier;
+import org.reactivestreams.Publisher;
+
+/**
+ * Optional slice that uses some source to create new slice
+ * if this source matches specified predicate.
+ * @param <T> Type of target to test
+ * @since 0.10
+ */
+public final class OptionalSlice<T> implements Slice {
```
Comment (olenagerasimova) [inline src/main/java/com/artipie/http/OptionalSlice.java:41]: @g4s8 same here - we need test

**Comment by g4s8** [comment]
@olenagerasimova thanks, fixed in https://github.com/artipie/artipie/pull/439/commits/8b6810ab484a62f7593b96f2392a33761e0b7a91

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#439` is not assigned, can't get performer

**Pull request merged**